apps/condo/domains/organization/gql.js

```diff
@@ -1,14 +1,24 @@
-const { gql } = require('graphql-tag')
-const { genTestGQLUtils } = require('@core/keystone/gen.gql.utils')
+/**
+ * Generated by `createschema organization.Organization 'country:Select:ru,en; name:Text; description?:Text; avatar?:File; meta:Json; employees:Relationship:OrganizationEmployee:CASCADE; statusTransitions:Json; defaultEmployeeRoleStatusTransitions:Json' --force`
+ * In most cases you should not change it by hands
+ * Please, don't remove `AUTOGENERATE MARKER`s
+ */
 
-const ORGANIZATION_FIELDS = '{ id dv sender v country name description avatar { publicUrl } statusTransitions defaultEmployeeRoleStatusTransitions createdBy { id } }'
-const Organization = genTestGQLUtils('Organization', ORGANIZATION_FIELDS)
+const gql = require('graphql-tag')
 
-const ORGANIZATION_ROLE_FIELDS = '{ id dv sender v name statusTransitions canManageOrganization canManageEmployees canManageRoles canManageProperties canManageTickets }'
-const OrganizationEmployeeRole = genTestGQLUtils('OrganizationEmployeeRole', ORGANIZATION_ROLE_FIELDS)
+const { generateGqlQueries } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+
+const COMMON_FIELDS = 'id dv sender v createdBy { id name } updatedBy { id name } createdAt updatedAt'
+
+const ORGANIZATION_FIELDS = `{ country name description avatar { publicUrl } meta statusTransitions defaultEmployeeRoleStatusTransitions ${COMMON_FIELDS} deletedAt }`
```

Comment:
deletedAt is already in COMMON_FIELDS

Reply:
No, because `OrganizationEmployee` and `OrganizationEmployeeRole` does not have it.

Reply:
It's better no to use COMMON_FIELDS for Organization and OrganizationEmployeeRole

Reply:
Should I use it like this?
What the reason? Automatic code transformation in the future?

```
const COMMON_FIELDS = 'id dv sender v deletedAt newId createdBy { id name } updatedBy { id name } createdAt updatedAt'

const ORGANIZATION_FIELDS = `{ country name description avatar { publicUrl } meta statusTransitions defaultEmployeeRoleStatusTransitions ${COMMON_FIELDS} }`
const Organization = generateGqlQueries('Organization', ORGANIZATION_FIELDS)

const ORGANIZATION_EMPLOYEE_ROLE_FIELDS = `{ organization { id } name statusTransitions canManageOrganization canManageEmployees canManageRoles canManageIntegrations canManageProperties canManageTickets id dv sender v createdBy { id name } updatedBy { id name } createdAt updatedAt }`
const OrganizationEmployeeRole = generateGqlQueries('OrganizationEmployeeRole', ORGANIZATION_EMPLOYEE_ROLE_FIELDS)

const ORGANIZATION_EMPLOYEE_FIELDS = `{ organization ${ORGANIZATION_FIELDS} user { id name } name email phone role ${ORGANIZATION_EMPLOYEE_ROLE_FIELDS} isRejected isAccepted id dv sender v createdBy { id name } updatedBy { id name } createdAt updatedAt }`
const OrganizationEmployee = generateGqlQueries('OrganizationEmployee', ORGANIZATION_EMPLOYEE_FIELDS)
```

Reply:
Changed in `6453bf1c`